app/utils/mail_carrier.rb

```diff
@@ -0,0 +1,15 @@
+module MailCarrier
+
+  module_function
+
+  # This is just a placeholder. Delivering later
+  # hasn't been implemented you.
+  def deliver_later(message)
+    deliver_now(message)
+  end
+
+  def deliver_now(message)
+    message.deliver_now
+  end
```

Comment:
Having only one method `deliver` was indeed my original idea too. However, there are cases where we probably want to send the email right away, e.g. from a worker job.